cb_hr_views/views/hr_attendance_module_views.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
     <record model="ir.ui.menu" id="hr_attendance.menu_hr_attendance_my_attendances">
-        <field name="groups_id" eval="[(6, 0, [ref('make_invisible')])]"/>
+        <field name="groups_id" eval="[(6, 0, [ref('manual_attendance_group')])]"/>
     </record>
     <record model="ir.ui.menu" id="hr_attendance.menu_hr_attendance_kiosk_mode">
         <field name="groups_id" eval="[(6, 0, [ref('make_invisible')])]"/>
```

Comment:
Remove make_invisible and set active=False

Reply:
@jarroyomorales

Reply:
Se me olvidó hacer push